xivo_test_helpers/asset_launching_test_case.py

```diff
@@ -12,11 +12,11 @@
 import docker as docker_client
 
 logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
 
-if os.environ.get('TEST_LOGS') != 'verbose':
-    logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARNING)
-    logging.getLogger('docker.auth').setLevel(logging.INFO)
-    logger.setLevel(logging.WARNING)
+logging.getLogger('urllib3.connectionpool').setLevel(logging.WARNING)
+logging.getLogger('docker').setLevel(logging.INFO)
+logger.setLevel(logging.DEBUG)
```

Comment:
Can you explain in a separate commit why these modifications?
* removing TEST_LOGS?
* renaming logger?

Reply:
I'll revert that. It was just to see logs in the job.

Reply:
The removal of TEST LOGS will be in standby until everyone as been consulted
the reason of renaming logger has been added to the commit message
Fixed

Reply:
Well there are a big difference with and without TEST_LOGS. So for now, I propose to keep it
https://zuul.wazo.community/paste/show/44/